tracing-core/src/field.rs

```diff
@@ -854,6 +858,10 @@ mod test {
         fn metadata(&self) -> &Metadata<'_> {
             &TEST_META_1
         }
+
+        fn registration(&'static self) -> &'static crate::callsite::Registration {
+            unimplemented!()
```

Comment:
nit: would be nice if this had a message indicating that this test should never call this method...